test/unit/specs/connectors/lcdClientMock.spec.js

```diff
@@ -144,7 +144,7 @@ describe(`LCD Client Mock`, () => {
   })
 
   it(`query and update the nonce`, async () => {
-    const { sequence } = await client.queryAccount(lcdClientMock.addresses[0])
```

Comment:
we didn't want to change the mock anymore

Reply:
I just find & replace, do you want me to undo those changes ?

Reply:
no, just want to raise awareness